src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const root = document.querySelector('body');
+
+const promise1 = new Promise((resolve) => (
+  logo.addEventListener('click', () => resolve())
+));
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => reject(new Error()), 3000);
+});
```

Comment:
will be better to create names according to the promise result: one of them should be the success one, other one should be the rejected one